.pipelines/pipeline.user.windows.yml

```diff
@@ -90,6 +91,17 @@ build:
             - 'modules\launcher\Plugins\Microsoft.Plugin.WindowWalker\Microsoft.Plugin.WindowWalker.dll'
             - 'modules\launcher\Plugins\Microsoft.Plugin.WindowWalker\Wox.Infrastructure.dll'
             - 'modules\launcher\Plugins\Microsoft.Plugin.WindowWalker\Wox.Plugin.dll'
+            - 'modules\launcher\PowerLauncher.dll'
+            - 'modules\launcher\PowerLauncher.exe'
+            - 'modules\launcher\PowerLauncher.Telemetry.dll'
+            - 'modules\launcher\PowerLauncher.UI.exe'
+            - 'modules\launcher\PowerToysInterop.dll'
+            - 'modules\launcher\Telemetry.dll'
+            - 'modules\launcher\Wox.Core.dll'
+            - 'modules\launcher\Wox.dll'
+            - 'modules\launcher\Wox.exe'
```

Comment:
I guess we dont include wox.exe because of #3068 ?

Reply:
good catch.  I"ll remove